xbmc/pvr/PVRManager.h

```diff
@@ -99,7 +99,7 @@ namespace PVR
      * @brief Updates the last watched timestamps of the channel and group which are currently playing.
      * @param channel The channel which is updated
      */
-    void UpdateLastWatched(CPVRChannel &channel);
+    void UpdateLastWatched(const CPVRChannelPtr &channel);
```

Comment:
this method is supposed to update the members in the channel, how can it be const?